llvm/test/MC/Disassembler/AMDGPU/vinterp.txt

```diff
@@ -0,0 +1,723 @@
+; NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Please do the conversion of the test to autogenerated and removal of the old vinterp-fake16.s as a pre-commit, before the functional changes.

Reply:
done